deploy/scripts/src/btm/test.ts

```diff
@@ -0,0 +1,60 @@
+import { type Options } from 'k6/options'
+
+import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
+
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js'
+import { AWSConfig, SQSClient } from 'https://jslib.k6.io/aws/0.7.2/sqs.js'
+
+const profiles: ProfileList = {
+  smoke: {
+    sendEventSmokeTest1: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 2,
+      stages: [
+        { target: 1, duration: '5s' } // Ramps up to target load
+      ],
+      exec: 'sendEvent'
+    }
+  }
+}
+
+const loadProfile = selectProfile(profiles)
+
+export const options: Options = {
+  scenarios: loadProfile.scenarios,
+  thresholds: {
+    http_req_duration: ['p(95)<=1000', 'p(99)<=2500'], // 95th percentile response time <=1000ms, 99th percentile response time <=2500ms
+    http_req_failed: ['rate<0.05'] // Error rate <5%
+  }
+}
+
+export function setup (): void {
+  describeProfile(loadProfile)
+}
+
+const env = {
+  sqs_queue: __ENV.BTM_AWS_SQS_QUEUE
+}
+
+const awsConfig = new AWSConfig({
+  region: __ENV.BTM_AWS_REGION,
+  accessKeyId: __ENV.BTM_AWS_ACCESS_KEY_ID,
+  secretAccessKey: __ENV.BTM_AWS_SECRET_ACCESS_KEY,
+  sessionToken: __ENV.BTM_AWS_SESSION_TOKEN
```

Comment:
Is the plan here to use temporary credentials to test?

I think longer term we need to look at a better way of delegating access or assume roles for accessing SQS queues

Reply:
As discussed, yes to support the K6 AWS Lib plan to use temp credentials and look at a longer term solution to automate this step